diff -urN aegisub-2.1.8.orig/src/main.cpp aegisub-2.1.8/src/main.cpp
--- aegisub-2.1.8.orig/src/main.cpp	2011-01-12 22:50:58.597584491 +0300
+++ aegisub-2.1.8/src/main.cpp	2011-01-12 23:02:38.334524833 +0300
@@ -137,11 +137,13 @@
 #endif
 #endif
 
+#if wxUSE_ON_FATAL_EXCEPTION
 		// Crash handling
 #if !defined(_DEBUG) || defined(WITH_EXCEPTIONS)
 		StartupLog(_T("Install exception handler"));
 		wxHandleFatalExceptions(true);
 #endif
+#endif
 
 		// Set config file
 		StartupLog(_T("Load configuration"));
